testutil/keeper/authority.go

```diff
@@ -74,8 +75,13 @@ func AuthorityKeeper(t testing.TB) (*keeper.Keeper, sdk.Context) {
 }
 
 // MockIsAuthorized mocks the IsAuthorized method of an authority keeper mock
-func MockIsAuthorized(m *mock.Mock, address string, policyType types.PolicyType, isAuthorized bool) {
-	m.On("IsAuthorized", mock.Anything, address, policyType).Return(isAuthorized).Once()
+// TODO : https://github.com/zeta-chain/node/issues/2153
```

Comment:
We should add a sentence explaining todo in addition to the link

Reply:
d3741a6e5e58747e2fa9cd6f39ece7d6ca0f0a37